golden_response/staging/get_landing_page/county.json

```diff
@@ -470,36 +470,372 @@
     },
     "Count_Person_InLaborForce": {
       "data": {
-        "1990": 852348,
-        "1991": 841771,
-        "1992": 847030,
-        "1993": 847518,
-        "1994": 851768,
-        "1995": 859592,
-        "1996": 891646,
-        "1997": 930075,
-        "1998": 952455,
-        "1999": 957792,
-        "2000": 941470,
-        "2001": 938612,
-        "2002": 886631,
-        "2003": 843720,
-        "2004": 821616,
-        "2005": 815361,
-        "2006": 821565,
-        "2007": 843437,
-        "2008": 868908,
-        "2009": 874982,
-        "2010": 926525,
-        "2011": 939851,
-        "2012": 958281,
-        "2013": 971863,
-        "2014": 989909,
-        "2015": 1012212,
-        "2016": 1027805,
-        "2017": 1038698,
-        "2018": 1043641,
-        "2019": 1053723
+        "1990-01": 847597,
```

Comment:
Ah... we should probably fix the obs period to avoid these diffs?

Reply:
Hmm.. sees like the returned data switched from yearly to monthly. Did any ranking change? Or is it the new cache?

Reply:
I think this is from the filter method which prefers BLS-LAUS mmethod

Reply:
Never mind, I didn't read Bo's comment about it.

I think the obs period LG then.